### DISCUSSION TITLE
Does the fullscreen option work the same in windows chrome?

### DISCUSSION THREAD

**Discussion by startjava** [videojs/video.js#8673] [Q&A]
### Description

```
fullscreen: {options: {navigationUI: 'hide'}},

fullscreen: {options: {navigationUI: 'show'}},
```

i test result，hide and show value，windows chrome run effect same.

Is this option only for mobile devices

### Reduced test case

?

### Steps to reproduce

1.
2.
3.

### Errors

?

### What version of Video.js are you using?

last version

### Video.js plugins used.

?

### What browser(s) including version(s) does this occur with?

chrome

### What OS(es) and version(s) does this occur with?

?

**Comment by mister-ben** [comment]
It's up to the browser how to respond to that. `navigationUI` is an indication of the preferred behaviour, but a browser doesn't have to honour it. My observation is also that Chrome doesn't do anything differently, at least for a regular browser scenario.
